database/tables/enrollments.pg

```diff
@@ -2,13 +2,23 @@ columns
     course_instance_id: bigint not null
     created_at: timestamp with time zone default CURRENT_TIMESTAMP
     id: bigint not null default nextval('enrollments_id_seq'::regclass)
-    user_id: bigint not null
+    lti_synced: boolean not null default false
+    pending_uid: text
+    status: enum_enrollment_status not null default 'joined'::enum_enrollment_status
+    user_id: bigint
 
 indexes
     enrollments_pkey: PRIMARY KEY (id) USING btree (id)
+    enrollments_pending_uid_course_instance_id_idx: UNIQUE USING btree (pending_uid, course_instance_id)
+    enrollments_user_id_course_instance_id_idx: UNIQUE USING btree (user_id, course_instance_id)
```

Comment:
You appear to have added a duplicate index? See the existing line immediately after this.